guides/standards-assurance-tables.md

```diff
@@ -0,0 +1,158 @@
+
+Using the [categories used within the MCSS](https://ministryofjustice.github.io/security-guidance/principles/identify-protect-detect-respond-recover/#identify-protect-detect-respond-recover) as sections some helpful thought questions and documentation expectations are discussed below.
+
+### IDENTIFY
+
+#### Possible documentation
+
+- Team organisation charts
+- Data Privacy Impact Assessments (DPIAs)
+- Information risk management documentation (for example, RMADS)
+- Information flow diagrams
+
+#### Thought questions
+
+- Who is responsible and/or accountable for the the system whether from an operational or budgetary perspective?
+- Who is responsible and/or accountable for the information held inside the system?
+- What security-focused work has been conducted on any suppliers and supplier systems to ensure they are safe for use/integration?
+- Where is the system technically hosted?
+- Where are the places where the system stores data?
+- What are the consequences if the system is unavailable to users or data has been lost/corrupted?
+- What happens for business continuity / disaster recovery if the system is unavailable or data has been lost/corrupted?
+
+### PROTECT
+
+#### Possible documentation
+
+- Data Privacy Impact Assessments (DPIAs)
+- Information risk management documentation (for example, RMADS)
+- Information flow diagrams
+- Technical/system architecture documentation
+- Penetration test / IT Health Check reports and remedial documentation
+- Risk registers
+
+#### Thought questions
+
+- How does the system ensure only authorised people can use the system?
+- How are system users managed for joiners, movers and leavers?
+- How is the system's underlying software kept up to date for security software patching?
+- How does the system protect itself appropriately and proportionately from attackers?
+- How do ensure the system can protect itself from attackers over time, so it is secure now but also will remain secure in the future?
+- How often has technical security testing been conducted? Where within the system?
+- How does the system stay up to date using modern encryption to keep data safe?
+- Does the system use multi-factor authentication (MFA, also known as 2FA)?
+- For people who have access to the system, do they have all the right clearances in place?
+
+### DETECT
+
+#### Possible documentation
+
+- Information risk management documentation (for example, RMADS)
+- Technical/system architecture documentation
+- Penetration test / IT Health Check reports and remedial documentation
+- Risk registers
+
+#### Thought questions
+
+- How does the system, and accompanying operational support teams, know/detect when the system is under attack?
+- How is access to the system (both legitimate and illegitimate) logged so retrospective investigations can take place to determine 'who did what when'?
```

Comment:
Add:
How is the required level of detail in logs determined? How long are log files retained?